README.md

```diff
@@ -57,6 +57,9 @@ kontent environment add --name DEV --api-key <Api key> --project-id <Project ID>
 npm run migrate 01_sample_init_createBlogType
 ```
 
+Kontent-CLI supports only running javascript migration files so in case you want to write in TypesSript, CoffeScript or in any other language you have to transpile your code before running.
+In the case of typescript, you may use this example from [Kontent CLI boilerplate](https://github.com/Kentico/kontent-migrations-boilerplate/blob/master/package.json#L7)

```

Comment:
As in the previous comment, I'd suggest using spellcheck and consistent capital letters so that you don't write TypesSript first and typescript second. Also, you could slightly rewrite this to say something like "If you're using TypeScript, see this example from Kontent CLI boilerplate." And I'd put the link on "example from KCB" so that people know you're linking to a specific example and not to KCD. People usually skim what's written and check link text.